src/chess/uciloop.cc

```diff
@@ -246,7 +246,8 @@ void UciLoop::SendInfo(const std::vector<ThinkingInfo>& infos) {
     if (info.game_id != -1) res += " gameid " + std::to_string(info.game_id);
     if (info.is_black)
       res += " side " + std::string(*info.is_black ? "black" : "white");
-    if (info.depth >= 0) res += " depth " + std::to_string(info.depth);
+    if (info.depth >= 0)
+      res += " depth " + std::to_string(std::max(info.depth, 1));
```

Comment:
I suppose if we want to keep these on one line…
```diff
+    if (info.depth == 0) res += " depth 1";
+    if (info.depth > 0) res += " depth " + std::to_string(info.depth);
```

Reply:
Maybe not report it at all when it's 0? I think cutechess can live with it..

Reply:
Well, I got reminded of this bug when looking at some pgn from test-results showing a missing mate score when there's a forced move. And similarly, the issue was originally filed because tcec showed an unreasonable 0.00 score and someone recently shared ccc showing 0.00 score when about to be mated.

I would think not reporting a depth of 0 would be the same behavior as reporting a depth of 0 and not fixing the issue.

Reply:
Doesn't build, seems that `#include <algorithm>` is missing.